File: Search_Insert_Position.py
#https://leetcode.com/problems/search-insert-position/description/

class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:
        n=len(nums)
        c=0
        if(target in nums):
            return nums.index(target)
        for i in range(n):
            if target<nums[i]:
                return i
                c=1
                break
        if (c==0):
            return n

File: Search_Insert_Position.cpp
//https://leetcode.com/problems/search-insert-position/description/

class Solution {
public:
    int searchInsert(vector<int>& nums, int target) {
        int c=0;
        for(int i=0;i<nums.size();i++){
            if(nums[i]==target) return i;
        }
        for(int i=0;i<nums.size();i++){
            if(target<nums[i]){
            return i;
            //c=1;
            //break;
            }
        }
        if(c==0){
        return nums.size();
        }
        return 0;
        
    }
};
